src/server/bg_services/log_replication_scanner.js

```diff
@@ -218,6 +238,36 @@ class LogReplicationScanner {
         dbg.log2('log_replication_scanner: scan delete_objects: ', keys);
     }
 
+    //TEMP This code is duplicated from Bucket diff, not cool, we need to place them together... 
```

Comment:
let's do it in this PR... as we will probably forget. just take it outside of BucketDiff, add a new export, and send AWS.S3 to it as well.